src/GameManipulator.php

```diff
@@ -203,12 +224,13 @@ public function switchCurrentPlayer($game): Game
      * 
      * @param Shoot $shoot
      * @param Game $game
+     * @param Player $player
      * 
      * @return Ship|null
      */
-    public function didShootHitOpponentShip(Shoot $shoot, Game $game): ?Ship
+    public function didShootHitOpponentShip(Shoot $shoot, Game $game, Player $player): ?Ship
```

Comment:
Better! But i'm still confused about which player it is? The current one? The opponent?
Using another name can fix this (eg: shootingPlayer)